go.mod

```diff
@@ -153,6 +153,8 @@ require (
 	k8s.io/klog/v2 v2.90.1
 )
 
+require github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws v0.24.0
```

Comment:
Dont think this needs a separate require block

Reply:
Where is this being used? I dont see any new imports from this in this PR.

Reply:
removed form the code.